man/Makefile.am

```diff
@@ -0,0 +1,123 @@
+#
+# This is a stand-alone Makefile for Universal-ctags maintainers.
+# GNU Make is needed.
+#
+# This doesn't work with Makefile.am at the top of source tree.
```

Comment:
Now this file works well with Makefile.am at the top of source tree:-)

Reply:
I've removed the last sentence.